CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [0.67.123](https://github.com/brave/brave-browser/releases/tag/v0.67.123)
+
+ - Added the ability to opt-in to user private Brave ads for Australia, New Zealand and Ireland. ([#5153](https://github.com/brave/brave-browser/issues/5153))
+ - Added the ability to tip Twitter publishers. ([#4346](https://github.com/brave/brave-browser/issues/4346))
```

Comment:
I would move tips to first place

Reply:
I debated between that and Ads. wdyt @kjozwiak ?

Reply:
why I said this is because we will have PR around twitter, but not around ads

Reply:
yes, makes sense. will change the order.